RunningOrder/AppDelegate.swift

```diff
@@ -12,28 +12,7 @@ import SwiftUI
 @NSApplicationMain
 class AppDelegate: NSObject, NSApplicationDelegate {
 
-    var window: NSWindow!
-
-
-    func applicationDidFinishLaunching(_ aNotification: Notification) {
-        // Create the SwiftUI view that provides the window contents.
-        let contentView = ContentView()
-
-        // Create the window and set the content view. 
-        window = NSWindow(
-            contentRect: NSRect(x: 0, y: 0, width: 480, height: 300),
-            styleMask: [.titled, .closable, .miniaturizable, .resizable, .fullSizeContentView],
-            backing: .buffered, defer: false)
-        window.center()
-        window.setFrameAutosaveName("Main Window")
-        window.contentView = NSHostingView(rootView: contentView)
-        window.makeKeyAndOrderFront(nil)
-    }
-
-    func applicationWillTerminate(_ aNotification: Notification) {
-        // Insert code here to tear down your application
-    }
-
+    func applicationDidFinishLaunching(_ aNotification: Notification) { }
 
+    func applicationWillTerminate(_ aNotification: Notification) { }
```

Comment:
are these two empty methods necessary?

Reply:
don't know but as it is the very 2 important methods, else we don't have any App Delegate !! it seems very weiiiiiiiird xD I preferred to keep this twos like this :)